Source/WebCore/html/parser/HTMLEntityParser.h

```diff
@@ -31,6 +31,7 @@
 namespace WebCore {
 
 bool consumeHTMLEntity(SegmentedString&, StringBuilder& decodedEntity, bool& notEnoughCharacters, UChar additionalAllowedCharacter = '\0');
+void appendLegalEntityFor(UChar32, StringBuilder& decodedEntity);
```

Comment:
The name `decodedEntity` is not a good name for a `StringBuilder` that we will be appending to. Sure, the thing we are appending is a decoded entity.

Reply:
Ok. Proposing `ouputBuffer`